### PR TITLE
 🔧 [#1629] Refactor settings module

### LINKED ISSUE #1629: As PO I want to streamline all environment variables.
### Thema / Theme

Other

### Omschrijving / Description

The end goal is to streamline the environment variables but first task is to create a comparison matrix.

* [x] Please make a matrix that includes all environment variables of Open Zaak, Open Notificaties, Open Klant, Objects and Objecttypes API. This document can be an internal one, or we can make it part of Open API framework.
* [ ] Bring other settings files in line with eachother
* [ ] #1649

After this, we can compare and decide how to proceed.

### Toegevoegde waarde / Added value

_No response_

### Aanvullende opmerkingen / Additional context

_No response_

PRs:
- [x] Open Klant https://github.com/maykinmedia/open-klant/pull/190
- [x] Open Notificaties https://github.com/open-zaak/open-notificaties/pull/163
- [ ] Open Zaak https://github.com/open-zaak/open-zaak/pull/1696
- [x] Objects https://github.com/maykinmedia/objects-api/pull/413
- [x] Objecttypes https://github.com/maykinmedia/objecttypes-api/pull/102
- [ ] Referentielijsten https://github.com/maykinmedia/referentielijsten/pull/17

### PR DESCRIPTION
**Pull request by stevenbal**
partially fixes #1629 

**Changes**

* Refactor settings module to use OAF base settings

Requires https://github.com/maykinmedia/open-api-framework/pull/35 to be merged and released

### CODE REVIEW

File: src/openzaak/conf/includes/base.py [hunk 1]
```diff
@@ -558,75 +179,13 @@
 #
 # DJANGO-AXES
 #
-AXES_CACHE = "axes"  # refers to CACHES setting
 AXES_FAILURE_LIMIT = 5  # Default: 3
```
Comment (SonnyBA) [inline src/openzaak/conf/includes/base.py:182]: I think this setting and the two below could also be removed (as they are in OAF already with the same values)

File: src/openzaak/conf/includes/base.py [hunk 2]
```diff
@@ -643,105 +202,21 @@
...
 # MAYKIN-2FA
 #
 # Uses django-two-factor-auth under the hood, so relevant upstream package settings
 # apply too.
 #
 
-# we run the admin site monkeypatch instead.
-TWO_FACTOR_PATCH_ADMIN = False
 # Relying Party name for WebAuthn (hardware tokens)
 TWO_FACTOR_WEBAUTHN_RP_NAME = "Open Zaak - admin"
```
Comment (SonnyBA) [inline src/openzaak/conf/includes/base.py:219]: I think we can remove this too, if we use `PROJECT_NAME.title()` in OAF

**Review by SonnyBA — CHANGES_REQUESTED** [review]
Two comments but overall looks good! :+1:

File: src/openzaak/conf/includes/base.py (hunk 1 above)
Comment (stevenbal) [reply src/openzaak/conf/includes/base.py:182]: You're right, forgot about those! I'll remove them

**Review by SonnyBA — APPROVED** [review]

File: requirements/base.txt [hunk 3]
```diff
@@ -175,7 +175,9 @@ django-rest-framework-condition==0.1.1
 django-sendfile2==0.7.0
     # via django-privates
 django-setup-configuration==0.1.0
```
Comment (annashamray) [inline requirements/base.txt:177]: I think you can now remove `django-setup-configuration` from base.in

**Review by annashamray — APPROVED** [review]
:+1:

File: requirements/base.txt (hunk 3 above)
Comment (stevenbal) [reply requirements/base.txt:177]: You're right! I also removed `djangorestframework-inclusions`

**Pull request merged**